tests/vmi_configuration_test.go

```diff
@@ -1290,6 +1292,7 @@ var _ = Describe("Configurations", func() {
 				_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(Vmi)
 				Expect(err).ToNot(HaveOccurred())
 				node1 := tests.WaitForSuccessfulVMIStart(Vmi)
+				// what should i do here it will always fail for us
 				Expect(node1).To(ContainSubstring("node02"))
```

Comment:
Instead of checking for the node, you can lookup the node it is scheduled on and check if the node has the `v1.CPUMangager: "true"` label.

Reply:
The same for the test above where we check if it is on node `node02`. There you can check if the node does not have that label.